src/Webprofiler/WebprofilerServiceProvider.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\rules\WebProfiler\WebprofilerServiceProvider.
+ */
+
+namespace Drupal\rules\WebProfiler;
+
+use Drupal\Core\DependencyInjection\ContainerBuilder;
+use Drupal\Core\DependencyInjection\ServiceProviderBase;
+
+/**
+ * Defines a service profiler for the web profiler module.
+ */
+class WebProfilerServiceProvider extends ServiceProviderBase {
```

Comment:
This has to be called RulesServiceProvider and must be placed into the Drupal\rules namespace to let Drupal find it